Share plotting code across conditions; pass in # of runs. Change std computation.

bcipolli opened the pull request:
@vishaalprasad a few changes I made while getting the plotting code running this morning.

Most important is the change in the `stddev` computation. I take the mean over trials first, then std over models. We can talk about why that's better.

vishaalprasad commented:
@bcipolli 
If I want to start working on my local repo and don't want any conflicts, do I just run a fetch from my origin?

bcipolli commented:
@vishaalprasad on another PR, I added instructions on how to check out the branch and start working from there. You should have that in your email somewhere.

Another way is to run `git merge [remote]/[branch name]` to get other code into your current branch. So for this branch, you can run `git merge bcipolli/fix-plot`. I had instructions on how to add the `bcipolli` remote in the PR referenced above.

Unfortunately, I couldn't find the PR (maybe an issue?) where I added that info... LMK if you can't find it either.
